atom/common/api/atom_api_shell.cc

```diff
@@ -12,12 +12,22 @@
 
 namespace {
 
+bool OpenExternal(const GURL& url, mate::Arguments* args) {
+  bool without_activation = false;
+  if (args->Length() == 2) {
+    mate::Dictionary options;
+    args->GetNext(&options);
```

Comment:
I think you may want to check the return value of `GetNext` and only use it when it returns `true`.

Currently doing `shell.openExternal('http://google.com', null)` on this pull request will crash the render process.